lmp/bb-config.sh

```diff
@@ -29,10 +29,10 @@ if [ "$ENABLE_PTEST" = "1" ] ; then
     OSTREE_BRANCHNAME="${OSTREE_BRANCHNAME}-ptest"
 fi
 
-if [ -n "$SOTA_PACKED_CREDENTIALS" ] && [ -f $SOTA_PACKED_CREDENTIALS ] ; then
+if [ -f "/secrets/targets.sec" ] ; then
```

Comment:
Will it work in the `else` case, I mean if the `targets.sec` secret is not available?

Reply:
yes. this is how people bitbake locally. meta-updater just says something about not being able to publish